db/migrate/20230329161439_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.string :Bio
```

Comment:
- Hi :clap: , the bio should be in lowercase and should be a text. `  t.text :bio`. let's kindly fix this please.